apache-rat-plugin/src/it/RAT-314/verify.groovy

```diff
@@ -21,7 +21,7 @@ assert content.contains( 'BUILD SUCCESS' )
 /*
  * [WARNING] No resources included
  */
-assert ! content.contains( '[WARNING]' )
+assert ! content.contains( '[WARNING] No resources' )
```

Comment:
Would we want to check for "[WARNING] No resources included" as stated in the comment above the assertion? or change the comment?

Reply:
Fixed.